website/src/pages/case-studies/lyft-monolith-to-microservices.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
+import { CaseStudyPage, InContentBlockquote, CaseStudyRequestDemoForm } from '../../components/content/CaseStudyPage'
+import { ContentSection } from '../../components/content/ContentSection'
+import Layout from '../../components/Layout'
+
+const justinPhilips = 'Justin Phillips, Software Engineer, Lyft'
+export default ((props: any) => (
+    <Layout
+        location={props.location}
+        meta={{
+            title:
+                'How Sourcegraph ensured production stability at Lyft during their monolith to microservices decomposition',
+            description:
+                'Learn how Sourcegraph code search helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition',
```

Comment:
```suggestion
                'Learn how Sourcegraph code search helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition',
```

Reply:
The word "free" is supporting the word "issue" so I believe the usage of a hyphen here correct.